runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -202,6 +202,10 @@ impl<T: Config> ProvideInherent for Module<T> {
 					{
 						Call::inclusion(signed_bitfields, backed_candidates, parent_header)
 					} else {
+						sp_runtime::print(
+							"WARN: dropping signed_bitfields and backed_candidates because they produced \
+							an invalid inclusion inherent. Probably this is related to a session change."
```

Comment:
Hm, I am not yet sure if we can say if that's the case

Reply:
Yeah, let's either avoid speculating in this message or give a different indication when the current block is the first block of the session (`Scheduler::SessionStartBlock`) might help.

Reply:
6e5bb03 avoids speculation. eeb79ab has some (broken) work towards adding a note when the current block is the first block of the session, but it's not compiling for reasons which seem tedious to work through. I think it's not worth it, so I reverted that commit.